sync/class.jetpack-sync-actions.php

```diff
@@ -36,15 +48,14 @@ static function init() {
 			return;
 		}
 		
-		require_once dirname( __FILE__ ) . '/class.jetpack-sync-client.php';
-
-		self::$client = Jetpack_Sync_Client::getInstance();
+		require_once dirname( __FILE__ ) . '/class.jetpack-sync-sender.php';
+		self::$sender = Jetpack_Sync_Sender::getInstance();
 
 		// bind the do_sync process to shutdown
-		add_action( 'shutdown', array( self::$client, 'do_sync' ) );
+		add_action( 'shutdown', array( self::$sender, 'do_sync' ) );
 
 		// bind the sending process
-		add_filter( 'jetpack_sync_client_send_data', array( __CLASS__, 'send_data' ), 10, 2 );
+		add_filter( 'jetpack_sync_send_data', array( __CLASS__, 'send_data' ), 10, 2 );
```

Comment:
I kept it out of there so we could hook an alternative sender implementation into it during tests. I think it should probably stay where it is for now.